common/common-api/build.gradle

```diff
@@ -1,5 +1,6 @@
 dependencies {
     compile 'commons-codec:commons-codec:1.10'
+    compile 'org.apache.httpcomponents:httpclient:4.5.2'
```

Comment:
is this lib going to create some conflict with other projects?

Reply:
I double checked that, it's basically the lib which is used in project common-http.